src/Mutator/Number/AbstractNumberMutator.php

```diff
@@ -48,11 +48,13 @@
  */
 abstract class AbstractNumberMutator implements Mutator
 {
-    private const SLEEP_FUNCTION_NAMES = [
+    private const UNWANTED_TO_MUTATE_FUNCTIONS = [
         'sleep',
         'usleep',
         'time_nanosleep',
         'set_time_limit',
+        'memory_limit',
```

Comment:
this is also done by ini_set: `ini_set('memory_limit', '2048M');`

Reply:
I've added a case for `ini_set`, or what do you mean? could you please add an example of what should be ignored in mutations?

Reply:
`ini_set('memory_limit', -1);` should not be mutated.